README.md

```diff
@@ -198,6 +198,98 @@ To add copyright headers to all files with IntelliJ
 2. Create a new copyright profile matching existing copyright header found on all files, name it Dockstore (Settings -> Copyright -> Copyright Profiles -> Add New)
 3. Set the default project copyright to Dockstore (Settings -> Copyright)
 
+### Setting up a Mac with Apple silicon for Dockstore development
```

Comment:
Suggest "Setting up a Mac for Dockstore development"

Reply:
done